### PR TITLE
Main nav

### PR DESCRIPTION
**Pull request by realalexhomer**
Adds the md widget for left handed navigation.

### CODE REVIEW

**Comment by realalexhomer** [comment]
@jshor no need to review this yet I'm going to make some changes

**Comment by jshor** [comment]
@realalexhomer is this ready for review?

File: client/vivace/main-nav/template.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+			<md-button md-menu-origin class="main-nav-button" ng-click="$mdOpenMenu($event)">
+				<md-icon>filter_list</md-icon>
+				<label class="main-nav-icon-label">
+					Filters
+				</label>
+			</md-button>
+			<md-menu-content width="3" class="main-nav-menu-body">
+				<md-menu-item>
+					<md-button>
+						Filter 1
```
Comment (jshor) [inline client/vivace/main-nav/template.html]: are these for filtering the list w.r.t. fire/archive/my candidates?

**Comment by jshor** [comment]
LGTM

File: client/vivace/main-nav/template.html (hunk 1 above)
Comment (realalexhomer) [inline client/vivace/main-nav/template.html]: Yeah we will have to implement that next I think

**Pull request merged**